src/demos/list.html

```diff
@@ -35,123 +35,125 @@ <h1 style="margin: 0 auto; text-align: center">List</h1>
 
       <!-- simple list -->
       <div class="parent">
-        <div class="child right-aligned-text">simple list</div>
+        <div class="child right-aligned-text">simple list single selection</div>
```

Comment:
Do the usage files also need to be updated?

Reply:
Yeah, I think we'll need more usage examples. Can we handle this in a separate PR? Maybe a PE could do this as it would help with testing too?

Reply:
I was hoping a PE could review the doc I come up with and create usage examples.

Reply:
@driskull Would love to take this on when we're ready to go. Created an issue in the docs repo for tracking this effort. 👍🏻

Reply:
awesome! 👍 🥳